doc/changelog.qbk

```diff
@@ -12,7 +12,7 @@ Released -
 
 * Allow auto-deducing signature in `MOCK_METHOD_(NON_)CONST`
 * Replaced Boost facilities with std:: equivalents where existing in C++14
-* Deprecated MOCK_FUNCTOR_TPL as no longer required, use the non _TPL variant even for templates
+* Removed MOCK_*_TPL as they are no longer required, use the non _TPL variant even for templates
```

Comment:
We're deprecating and removing these in the same release?

Reply:
Yes. The switch to C++14 is already a breaking change and the "fix" for this is easy: Remove `_TPL` from your code base, nothing more. ;)
Also I'm just being a bit more explicit here: There wasn't a deprecation of `MOCK_FUNCTOR_TPL`, but it was replaced by a (failing) `static_assert`.

So yeah, basically preparing a new major release here and taking that liberty to reduce the amount of macros.

Reply:
I'm not upset about it, and don't use _TPL in my code.
I just wanted to make sure you were aware that there hadn't been a release with _TPL deprecated.